deep_speech_2/examples/librispeech/run_infer.sh

```diff
@@ -21,9 +21,10 @@ python -u infer.py \
 --num_conv_layers=2 \
 --num_rnn_layers=3 \
 --rnn_layer_size=2048 \
---alpha=0.36 \
```

Comment:
Please also update this in `examples/tiny`.

Reply:
Done